include/onemkl/blas/detail/cublas/blas_ct.hpp

```diff
@@ -4018,6 +4183,7 @@ cl::sycl::event symv<library::cublas, backend::nvidiagpu>(
     auto done = onemkl::cublas::symv(queue, upper_lower, n, alpha, a, lda, x, incx, beta, y, incy,
                                      dependencies);
     symv_postcondition(queue, upper_lower, n, alpha, a, lda, x, incx, beta, y, incy, dependencies);
+    done.wait();
```

Comment:
Could you please explain why we need to wait here?
And if synchronization is required why do we return the event?

Reply:
This is due to this issue (https://github.com/intel/llvm/issues/1569). 
However, I just found out that, the issue is fixed in recent builds, so I will test and update the code. (remove the wait().

Reply:
Removed the done.wait() command.